database/src/lib.rs

```diff
@@ -472,7 +472,7 @@ pub struct Index {
     artifacts: Indexed<Box<str>>,
     /// Id lookup of compile stat description ids
     /// For legacy reasons called `pstat_series` in the database, and so the name is kept here.
-    pstat_series: Indexed<(Benchmark, Profile, Scenario, CodegenBackend, Metric)>,
+    pstat_series: Indexed<(Benchmark, Profile, Scenario, CodegenBackend, Metric, String)>,
```

Comment:
Let's at least create a type alias `CompileTarget` in the database crate (in this file), to avoid having just `String` here, that's non-obvious.